scripts/install-zenml-dev.sh

```diff
@@ -48,11 +48,16 @@ install_integrations() {
     zenml integration export-requirements \
         --output-file integration-requirements.txt \
         $ignore_integrations_args
+
+    # pin pyyaml>=6.0.1
+    echo "" >> integration-requirements.txt
```

Comment:
It already failed during the pip install before the langchain integration is even installed

Reply:
(Since some other integrations use pyyaml too)

Reply:
It seemed to me that if you don't explicitly pin PyYAML globally, pip tries to download older versions of it if only to figure out if it can be used. I don't really understand why though.